Build/BUILDsis2

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#***********************************************************************
+#*                   GNU Lesser General Public License
+#*
+#* This file is part of the SHiELD Build System.
+#*
+#* The SHiELD Build System free software: you can redistribute it
+#* and/or modify it under the terms of the
+#* GNU Lesser General Public License as published by the
+#* Free Software Foundation, either version 3 of the License, or
+#* (at your option) any later version.
+#*
+#* The SHiELD Build System distributed in the hope that it will be
+#* useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+#* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#* See the GNU General Public License for more details.
+#*
+#* You should have received a copy of the GNU Lesser General Public
+#* License along with theSHiELD Build System
+#* If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+#  DISCLAIMER: This script is provided as-is and as such is unsupported.
+#
+set -x
+#
+# set default values
+# configure your build parameters
+  COMPILER="intel"
+  BIT="64bit"
+#
+# parse arguments
+  for arg in "$@"
+  do
+      case $arg in
+          intel|gnu|nvhpc)
+             COMPILER="${arg#*=}"
+             shift # Remove "compiler" from processing
+             ;;
+	  32bit|64bit)
+             BIT="${arg#*=}"
+             shift # Remove *bit from processing
+             ;;
+          *)
+          if [ ${arg#} != '--help' ] && [ ${arg#} != '-h' ] ; then
+            echo "option "${arg#}" not found"
+          fi
+          echo -e ' '
+          echo -e "valid options are:"
+          echo -e "\t[intel(D) | gnu | nvhpc] \t\t\t compiler"
+	  echo -e "\t[32bit(D) | 64bit] \t\t\t FV3 precision option"
+          echo -e "\n"
+          exit
+          ;;
+      esac
+  done
+
+#
+# set up some default variables if not called from COMPILE
+# BUILD_ROOT is set if this script is called from the COMPILE script
+if [ -z ${BUILD_ROOT} ] ; then
+   export BUILD_ROOT=${PWD%/*}
+   export SHiELD_SRC=${PWD%/*/*}/SHiELD_SRC/
+   export PATH="${BUILD_ROOT}/mkmf/bin:${BUILD_ROOT}/Build/mk_scripts:${PATH}"
+   export NCEP_DIR=${BUILD_ROOT}/Build
+   if [ ! -z ${EXTERNAL_LIBS} ] ; then
+      export NCEP_DIR=${EXTERNAL_LIBS}
+   fi
+   # load the proper environment for your machine
+   . ${BUILD_ROOT}/site/environment.${COMPILER}.sh
+fi
+
+export mkmf_template="/ncrc/home2/fms/local/opt/fre-commands/bronx-20/site/ncrc5/intel-classic.mk" #GAEA
```

Comment:
Duplicate Comment: Have you tried building mom6 and sis2 with the intel.mk in SHiELD_build/site/intel.mk? 

If that does work you can delete these export mkmf_template lines, and replace any reference to mkmf_template with `$BUILD_ROOT/$TEMPLATE` similarly to how other SHiELD components do.
https://github.com/NOAA-GFDL/SHiELD_build/blob/41662e11510c5c808eccf132a2ba9d167bdbfaed/Build/mk_scripts/MAKE_libFMS#L82

 If that doesn't work and mom6 and sis2 need a separate template file, I would recommend using the template in the mkmf directory (it is identical to the one referenced here) `/mkmf/templates/ncrc5-intel-classic.mk`. And then I would also surround this by a condition so that this template is only used for intel and give another option for gnu.

Reply:
same as above